Sources/PlayingCard.swift

```diff
@@ -31,13 +31,13 @@ public func ==(lhs: PlayingCard, rhs: PlayingCard) -> Bool {
 extension PlayingCard: Comparable {}
 
 public func <(lhs: PlayingCard, rhs: PlayingCard) -> Bool {
-    return lhs.rank == rhs.rank ? lhs.suit == rhs.suit : lhs.rank < rhs.rank
+    return lhs.rank == rhs.rank ? lhs.suit < rhs.suit : lhs.rank < rhs.rank
```

Comment:
the original logic allows for A<K in some cases.
![image](https://cloud.githubusercontent.com/assets/6539412/14894176/437293e0-0d27-11e6-99ef-046595022177.png)